common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -290,9 +294,19 @@ def test_deferred_verification_enrollment(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect enrollment activated event
+        self.assertEqual(
```

Comment:
Yeah that sounds good to me. Separate PR?

I'm looking at the helper classes, and it could potentially go with the UniqueCourseTest -- at least the setup --, but I could also see it as a couple of generic helper methods. Preferences on that?